### PR TITLE
Update .editorconfig

### PR DESCRIPTION
**Pull request by mrkoneru1**

### CODE REVIEW

**Comment by mrkoneru1** [comment]
Please approve

**Review by ktandabany — APPROVED** [review]
Approved

File: .editorconfig [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # EditorConfig is awesome: http://EditorConfig.org
```
Comment (mrkoneru1) [inline .editorconfig:1]: Changes are good

**Review by mrkoneru1 — COMMENTED** [review]
review finished

**Pull request closed without merging**